g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/Interceptor/APIValidationInterceptor.java

```diff
@@ -35,9 +40,18 @@
 	private static Logger LOGGER = LoggerFactory.getLogger(APIValidationInterceptor.class);
 
 	private Map<String, Object> allowedListMap;
-	public APIValidationInterceptor(Map<String, Object> allowedListMap) {
+	private String clientRequestIdsStr;
+	private List<String> clientRequestIds;
+	public APIValidationInterceptor(Map<String, Object> allowedListMap, String clientReqIdsStr) {
 		super();
 		this.allowedListMap = allowedListMap;
+		this.clientRequestIdsStr = clientReqIdsStr;
+		try {
+			this.clientRequestIds = Arrays.asList(this.clientRequestIdsStr.split(ConstantsChar.COMMA));
```

Comment:
Are you sure there are multiple headers used to trace?

Reply:
in Saas environment, need to support multiple headers